.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,36 @@
+# See: https://google.github.io/oss-fuzz/getting-started/continuous-integration/
+
+name: "CIFuzz"
+
+on:
+  push:
+    branches: [ "master", branch/* ]
```

Comment:
Should `branch/*` be enclosed in quotes?

Reply:
Not needed, but I agree it looks weird. I just swapped to the list version.